usaspending_api/disaster/v2/views/recipient/loans.py

```diff
@@ -0,0 +1,49 @@
+import json
+import re
+from decimal import Decimal
+from typing import List
+
+from usaspending_api.disaster.v2.views.disaster_base import ElasticsearchLoansPaginationMixin
+from usaspending_api.disaster.v2.views.elasticsearch_base import ElasticsearchDisasterBase
+
+
+class RecipientLoansViewSet(ElasticsearchLoansPaginationMixin, ElasticsearchDisasterBase):
```

Comment:
Probably worth commenting here and in the other ViewSet for spending that its the Mixin that drives which columns are summed on and the sums returned by the ES query.